network-api/networkapi/buyersguide/templates/product_page.html

```diff
@@ -39,11 +39,11 @@
   </div>
 </div>
 
-{% with section_class_col="col-12 col-lg-10 offset-lg-1" section_class_padding_x="px-md-5" section_class_padding_y="py-md-5" %}
+{% with section_class="col-12 col-lg-10 offset-lg-1 p-md-5" %}
```

Comment:
They were for https://github.com/mozilla/foundation.mozilla.org/pull/2303/files#r237297122 Since wrappers got changed in this PR, all `section_class` are sharing the exactly same classes. Hence we don't need to break them into 3 different variables.

Reply:
@Pomax ^